tests/unit/components/sort-menu/SortMenu.spec.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { screen, fireEvent } from '@testing-library/react'
+import SortMenu from '~/components/sort-menu/SortMenu'
+import { expect, vi } from 'vitest'
+import { renderWithProviders } from '~tests/test-utils'
+
+const mockSetSearchParams = vi.fn()
+
+vi.mock('react-router-dom', async (importOriginal) => {
+  const module = await importOriginal()
+
+  return {
+    ...module,
+    useSearchParams: () => [{ get: vi.fn() }, mockSetSearchParams]
+  }
+})
+
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key) => {
+      switch (key) {
+        case 'findOffers.sortTitles.newest':
+          return 'Newest'
+        case 'findOffers.sortTitles.rating':
+          return 'Rating'
+        case 'findOffers.sortTitles.priceAsc':
+          return 'Price: Low to High'
+        case 'findOffers.sortTitles.priceDesc':
+          return 'Price: High to Low'
+        default:
+          return key
+      }
+    }
+  })
+}))
```

Comment:
We already have mock of this library in `tests\setup-tests.js`, this mock  simple returns the obtained string.
```suggestion
```